### PR TITLE
Oleksijt task7 program to read last n lines of a file

### PR DESCRIPTION
**Pull request by Oleksijt**
Task 7 completed

### CODE REVIEW

File: oleksijt/task7_program_to_read_last_n_lines_of_a_file.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+""" This script reads last n lines of a file. """
+
+
+def read_file_lines(file_name, num_lines):
+    with open(file_name, 'r', encoding='utf-8') as f:
+        file_lines = f.readlines()
+        return file_lines[len(file_lines) - num_lines:]
```
Comment (pivancy) [inline oleksijt/task7_program_to_read_last_n_lines_of_a_file.py]: Isn't it possible to make the same thing easier?

Comment (Oleksijt) [reply oleksijt/task7_program_to_read_last_n_lines_of_a_file.py]: Done.

File: oleksijt/task7_program_to_read_last_n_lines_of_a_file.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+""" This script reads last n lines of a file. """
+
+
+def read_last_file_lines(file_name, num_lines):
+    with open(file_name, 'r', encoding='utf-8') as f:
+        return list(f)[-num_lines:]
```
Comment (pivancy) [inline oleksijt/task7_program_to_read_last_n_lines_of_a_file.py]: What is `list(f)`?

Comment (Oleksijt) [reply oleksijt/task7_program_to_read_last_n_lines_of_a_file.py]: Function list(f) creates list with lines from file f.

Comment (pivancy) [reply oleksijt/task7_program_to_read_last_n_lines_of_a_file.py]: Do you find its obvious?

Comment (Oleksijt) [reply oleksijt/task7_program_to_read_last_n_lines_of_a_file.py]: Fixed.

**Comment by pivancy** [comment]
👍

**Pull request merged**